src/XenCert/XenCertCommon.py

```diff
@@ -274,15 +274,25 @@ def printCommand(argvs):
             pass
         else:
             if option == '-i':
-                temp_argvs[option_index+1] = ':'.join(hidePassword(temp_argvs[option_index+1].split(':'), 2))
+                temp_argvs[option_index+1] = ':'.join(getNoPasswordCmds(temp_argvs[option_index+1].split(':'), 2))
             else:
                 temp_argvs[option_index+1] = HIDDEN_PASSWORD
 
     for argv in temp_argvs:
         PrintToLog(argv)
         PrintToLog(' ')
 
-def hidePassword(cmd, password_index=-3):
+def getNoPasswordCmds(cmd, password_index=-3):
```

Comment:
Suggest to change to another name, e.g. getCmdsWithHiddenPassword

Reply:
native english expression :+1: